docs/developer/procedures/supported.rst

```diff
@@ -125,7 +127,6 @@ The one most likely to cause grief is the JUnit testing requirement.
 
 2. The test coverage expectation for the GeoTools library is 40%.
        
-   * Test Coverage measured and published to Module Matrix page
    * Coverage of 40% measured by a hudson control profile.
```

Comment:
In theory. I wonder if it is still enabled or used...
I don't see any standalone job that handles it, and can't see anything that stands out in the job configuration.